.github/workflows/check.yaml

```diff
@@ -95,7 +94,7 @@ jobs:
 
       - name: install external controller required packages
         run: |
-           sudo snap install charmcraft --channel 2.x/stable --classic
```

Comment:
switching charmcraft channel is something we should do separately:

- update charmcraft 2 -> 3 everywhere in the repo (including the workflows)
- update charmcraft.yaml to follow latest conventions
- test it
- port changes to the workflows to the automation repo

Reply:
Ok, let's do it separately. I will remove this change.

Reply:
Deleted, so resolved.